service/controller/internal/configmap/current_test.go

```diff
@@ -404,9 +404,8 @@ func Test_ConfigMap_GetCurrentState(t *testing.T) {
 				objs = append(objs, cc)
 			}
 
-			fakeTenantK8sClient := fake.NewSimpleClientset(objs...)
 			tenantService := &tenantMock{
-				fakeTenantK8sClient: fakeTenantK8sClient,
+				fakeTenantRestConfig: &rest.Config{},
 			}
```

Comment:
@xh3b4sd change above for this test is a consequence of https://github.com/giantswarm/tenantcluster/pull/22 - test is actually not working, mocking config doesn't result in a mocked client; @rossf7 is proposing to drop this particular test since whole configmap resource here will be removed with appification, still I wanted to ping you as feedback for the new simplified tenantcluster API design, that unit tests like these seem to be no longer possible, at least I haven't found straightforward workaround yet

Reply:
Kill it with fire.

Reply:
The test has been removed.